tests/event_test.py

```diff
@@ -346,6 +346,31 @@ def test_read_receipt_event(self):
         assert isinstance(event, ReceiptEvent)
         assert receipt in event.receipts
 
+    def test_read_receipt_event_bad_ts(self):
+        """Test reading an m_receipt event that has malformed data for one user.
+
+        @alice:example.com is a user using Synapse pre 0.99.3 with a
+        timestamp bug. We want to ignore her malformed value without losing
+        the receipt data from @bob:example.com
+        """
+        parsed_dict = TestClass._load_response("tests/data/events/receipt_invalid.json")
+        print("starting")
```

Comment:
Can we get rid of those prints? They get printed out only when the test fails, but I'm not sure they are telling us something useful even if the test fails.

Reply:
Whoops, those shouldn't've been committed. Updated in the new commit.